### PR TITLE
Edit button and spiner on login page

### PR DESCRIPTION
**Pull request by lvymetalik**

### CODE REVIEW

File: django_fido/static/django_fido/js/fido2.js [hunk 1]
```diff
@@ -62,6 +65,10 @@ function fido2SuccessAuthenticationCallback(assertion) {
 
 function fido2ErrorResponseCallback(error) {
     addFido2Error(error.message);
+    document.getElementById('submit-button').style.opacity = "1";
```
Comment (ziima) [inline django_fido/static/django_fido/js/fido2.js]: I'd remove this style change. It should be part the particular `closeFidoWindow` hook.

File: django_fido/static/django_fido/js/fido2.js [hunk 2]
```diff
@@ -41,6 +41,9 @@ function base64_to_bytes(base64) {
     for (var i = 0; i < len; i++) {
         bytes[i] = binary_string.charCodeAt(i);
     }
+    if (typeof activeFidoWindow === 'function') {
+        activeFidoWindow();
+    }
```
Comment (ziima) [inline django_fido/static/django_fido/js/fido2.js]: Hook is good enough, but it shouldn't be placed into a conversion function. I'd rather put it into `fido2*RequestCallback`s below.

Comment (lvymetalik) [reply django_fido/static/django_fido/js/fido2.js]: Hook is moved to fido2*RequestCallback

File: django_fido/static/django_fido/js/fido2.js (hunk 1 above)
Comment (lvymetalik) [reply django_fido/static/django_fido/js/fido2.js]: Removed

**Pull request merged**